### DISCUSSION TITLE
Unable to proceed with verification.

### DISCUSSION THREAD

**Discussion by nikifutaki** [community/community#54620] [GitHub Education]
### Hi everyone,

I enrolled in a course that allows me to obtain a university degree from April this year, and I tried to apply for Github Education, but I cannot proceed beyond uploading the certificate. When I upload it, it displays "Please upload proof of your current school affiliation". I have attempted to upload my university acceptance letter, admission notification, and student ID. What am I missing?

**Comment by hrishikesh-kocharekar-dev** [comment]
It sounds like you may not be submitting the correct documentation to prove your current school affiliation when applying for Github Education. Here are some suggestions on what to try:

Contact Github Education Support: If you are still having trouble, I would recommend reaching out to Github Education Support directly to see if they can assist you in verifying your school affiliation.

Check Github Education eligibility requirements: Make sure that your university and course are eligible for Github Education benefits. Github Education is available to students who are currently enrolled in a degree or diploma-granting course at an accredited university or college.

Ensure that your documentation is up-to-date: Double-check that the documents you are submitting are up-to-date and include the necessary information to prove your current school affiliation. This can include your full name, the name of your university or college, and the current academic term or year.

Try a different document: If the certificate or student ID is not being accepted, try uploading a different document that clearly shows your current school affiliation. For example, you could try uploading a current class schedule or a letter from your school's registrar's office.

**Comment by samus-aran** [comment]
Hi @nikifutaki  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
